OpenRA.Game/Server/Server.cs

```diff
@@ -743,20 +743,43 @@ public void DispatchOrdersToClients(Connection conn, int frame, byte[] data)
 				if (c != conn && c.Validated)
 					DispatchFrameToClient(c, from, frameData);
 
+			RecordOrder(frame, data, from);
+		}
+
+		void RecordOrder(int frame, byte[] data, int from)
+		{
 			if (recorder != null)
 			{
-				recorder.ReceiveFrame(from, frame, data);
+				recorder.ReceiveFrame(@from, frame, data);
 
 				if (data.Length > 0 && data[0] == (byte)OrderType.SyncHash)
 				{
 					if (data.Length == Order.SyncHashOrderLength)
 						HandleSyncOrder(frame, data);
 					else
-						Log.Write("server", $"Dropped sync order with length {data.Length} from client {from}. Expected length {Order.SyncHashOrderLength}.");
+						Log.Write("server",
+							$"Dropped sync order with length {data.Length} from client {@from}. Expected length {Order.SyncHashOrderLength}.");
 				}
 			}
 		}
 
+		public void DispatchServerOrdersToClients(Order order)
```

Comment:
Missed the "CreateGameSave" case on line 908 when converting the `DispatchOrders(null, 0, ...`s to this.